api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -3281,4 +3283,35 @@ public void mergePatients_shouldMoveAllObsWithSameHierarchy() throws Exception {
 		assertEquals(2, encounterService.getEncounter(57).getObsAtTopLevel(true).size());
 	}
 
+
+	/**
+	 * Gets the patient , then sees if it can get the patient identifier by the patient program as well
+	 *
+	 * @throws Exception
+	 */
+	@Test
+	public void getPatientIdentifiersByPatientProgram_shouldGetPatientsByIdentifierByProgram() throws Exception {
```

Comment:
you wanna change the method name from `getPatientIdentifiersByPatientProgram_shouldGetPatientsByIdentifierByProgram()`  to something like

`getPatientIdentifiersByPatientProgram_shouldReturnPatientsIdentifiersByPatientProgram() `

Reply:
@tendomart this seems fine to me the way how it is

Reply:
Okk